ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -201,6 +201,15 @@
   public static final String storeReadWriteEnableSizeThresholdPercentageDeltaName =
       "store.read.write.enable.size.threshold.percentage.delta";
 
+  /**
+   * Enables or disables dynamic sealing/unsealing of the BlobStore
```

Comment:
minor: can drop the "of the BlobStore". Also, maybe it makes sense to place this config above the threshold configs?

Reply:
Sure to both